web/src/components/vis/ScorePlot.vue

```diff
@@ -82,6 +82,19 @@ export default {
      *   labels: Object<string, Array<string>>
      * }}
      */
```

Comment:
Seems like this comment is supposed to go directly above `rawPoints` but has been separated by inserted code.

Reply:
See 914fe8a2d4496fa9e48dd6d90d7f1789f52442ec.